packages/models/prophecy_model/lib/src/prophecy.dart

```diff
@@ -4,6 +4,9 @@ import 'package:meta/meta.dart';
 
 part 'prophecy.g.dart';
 
+/// exact prophecy model that will be showed in the apps
+
+/// can be expanded in future
 enum ProphecyId {
```

Comment:
What about renaming this to `ProphecyType` for example?
Despite formally (Id is usually a unique ID that represents one instance of an entity) it suits here, Im still always looking for a number or string for an ID representation.

Reply:
I will do it